sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -94,6 +94,10 @@ def __init__(
         **kwargs: Any
     ) -> None:
         self._loop = kwargs.pop("loop", None)
+        if isinstance(credential, AzureSasCredential):
```

Comment:
I guess we need to bump the azure-core dependency in setup.py to the version that introduced AzureSasCredential?

Reply:
true